uint32 id
string Class
geometry_msgs/Vector3 real_pose
geometry_msgs/Vector3 velocity
int64[] bbox
string color
int64 saturation
int64 ilumination
string shape

